man/sd_bus_error.xml

```diff
@@ -138,234 +150,188 @@
-    <citerefentry project='man-pages'><refentrytitle>printf</refentrytitle><manvolnum>3</manvolnum></citerefentry>
-    format and corresponding arguments.
-    <structfield>name</structfield> will be generated from
+    if applicable, otherwise a name in the
+    <literal>System.Error.</literal> namespace will be generated. The
+    error number specified may either be passed in negative or
+    positive, the call will implicitly take the absolute value. The
+    call returns the negative error number passed in, for convenient
+    usage in <function>return</function> statements. This call might
+    fail due to lack of memory, in which case an
+    <constant>SD_BUS_ERROR_NO_MEMORY</constant> error is set instead,
+    and -ENOMEM returned.</para>
+
+    <para><function>sd_bus_error_set_errnof()</function> is similar to
+    <function>sd_bus_error_set_errno()</function>, but in addition to
+    <parameter>error</parameter>, takes a <citerefentry
+    project='man-pages'><refentrytitle>printf</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    format string and corresponding arguments.  The
+    <structfield>message</structfield> field will be generated from
     <parameter>format</parameter> and the arguments.</para>
 
-    <para><function>sd_bus_error_get_errno</function> will convert
-    <structname>e-&gt;name</structname> to an errno-like value using the
-    same rules as <function>sd_bus_error_set</function>.  If
+    <para><function>sd_bus_error_set_errnofv()</function> is similar to
+    <function>sd_bus_error_set_errnof()</function> but takes the
+    format string parameters as <citerefentry
+    project='man-pages'><refentrytitle>va_arg</refentrytitle><manvolnum>3</manvolnum></citerefentry>
+    parameter list.</para>
+
+    <para><function>sd_bus_error_get_errno()</function> converts the
+    <structfield>name</structfield> field of an error structure to an
+    <varname>errno</varname>-like (positive) value using the same
+    rules as <function>sd_bus_error_set()</function>.  If
     <parameter>e</parameter> is <constant>NULL</constant>, 0 will be
     returned.</para>
 
-    <para><function>sd_bus_error_copy</function> will initialize
+    <para><function>sd_bus_error_copy()</function> will initialize
     <parameter>dst</parameter> using the values in
     <parameter>e</parameter>. If the strings in
     <parameter>e</parameter> were set using
-    <function>sd_bus_set_error_const</function>, they will be shared.
-    Otherwise, they will be copied.</para>
+    <function>sd_bus_set_error_const()</function>, they will be shared.
+    Otherwise, they will be copied. Returns a converted
+    <varname>errno</varname>-like, negative error code.</para>
 
-    <para><function>sd_bus_error_is_set</function> will return
-    <constant>true</constant> if <parameter>e</parameter> is
+    <para><function>sd_bus_error_is_set()</function> will return
+    non-zero if <parameter>e</parameter> is
```

Comment:
"sd_bus_error_is_set() -> sd_bus_error_is_set" to fix Semaphore first build error!?